classes/Customer.php

```diff
@@ -265,6 +269,33 @@ public function add($autoDate = true, $nullValues = true)
         return $success;
     }
 
+    /**
+     * Adds current Customer as a new Object to the database.
+     *
+     * @param bool $autoDate Automatically set `date_upd` and `date_add` columns
+     * @param bool $nullValues Whether we want to use NULL values instead of empty quotes values
+     *
+     * @return bool Indicates whether the Customer has been successfully added
+     *
+     * @throws PrestaShopDatabaseException
+     * @throws PrestaShopException
+     */
+    public function addWs($autodate = true, $null_values = false)
+    {
+        if (CustomerCore::customerExists($this->email)) {
+            WebserviceRequest::getInstance()->setError(
+                500,
+                $this->trans('The email is already used, please choose another one',
+                array(), 'Admin.Notifications.Error'),
```

Comment:
According to [PSR-2](https://www.php-fig.org/psr/psr-2/#44-method-arguments):

> Argument lists MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one argument per line.

So arguments need to be split up like this:

```diff
-$this->trans('The email is already used, please choose another one',
-array(), 'Admin.Notifications.Error'),
+$this->trans(
+    'The email is already used, please choose another one',
+    [],
+    'Admin.Notifications.Error'
+),
```

Reply:
Maybe one rule  we should add or find in php-cs-fixer, because everything seems ok for it :)

Reply:
@eternoendless  I made the changes you suggested